Islandora/islandora_solr_search
# Add paging to RSS results

**JIRA Ticket**: (https://jira.duraspace.org/browse/ISLANDORA-2439)

# What does this Pull Request do?

Adds paging to Solr RSS results.

# What's new?
Adds additional `<link>` elements in the `<channel>` element that provide the next, previous, and first pages.

# How should this be tested?

* Enable the RSS secondary display profile in your Solr settings
* Run a search with a lot of results (e.g. an empty search) and click the RSS button
* Note the new links in the `<channel>` section
* Note that your search is limited to the number of results in your RSS settings
* Put the content of the "next" link into your browser
* Result should contain the next X results of your search

# Interested parties
@Islandora/7-x-1-x-committers
